pins/rsconnect/fs.py

```diff
@@ -1,6 +1,6 @@
 from dataclasses import asdict, dataclass, field, fields
 from pathlib import Path
-from typing import Sequence
+from typing import ClassVar, Sequence, Tuple
```

Comment:
This will need  `from __future__ import annotations` as well for the `|` in Python 3.8/3.9 🔮